packages/taquito-http-utils/src/taquito-http-utils.ts

```diff
@@ -50,8 +48,8 @@ export class HttpResponseError extends Error {
 export class HttpRequestFailed extends Error {
   public name = 'HttpRequestFailed';
 
-  constructor(public url: string, public innerEvent: any) {
-    super(`Request to ${url} failed`);
+  constructor(public errorDetail: string) {
```

Comment:
Having access to the url might be helpful for a user to locate where the error happens, is there a reason to remove it?

Reply:
the errorDetail actually exposes many other properties from axios, including `url`

Reply:
ok perfect